schema/swagger.yml

```diff
@@ -58,13 +58,22 @@ paths:
       - application/json
       responses:
         200:
-          description: Node constants
+          description: Node Status Response
```

Comment:
Node constants response

Reply:
Done.

Reply:
@Isabello `Node Constants Response` as you requested, or `Node Status Response`?

Reply:
This is node status, as its for endpoint `/node/status`